client/packages/programs/src/JsonForms/components/IdGenerator.tsx

```diff
@@ -270,7 +270,7 @@ const useUniqueProgramEnrolmentIdValidation = () => {
 
 const UIComponent = (props: ControlProps) => {
   const { label, path, data, visible, handleChange, uischema, config } = props;
-  const t = useTranslation(['patients', 'common']);
+  const t = useTranslation('dispensary');
```

Comment:
The confirmation modal "warning" message wasn't being rendered -- it's in the dispensary.json